server/plugin/api_test.go

```diff
@@ -941,82 +952,37 @@ func TestHandlePipelineApproveOrRejectRunRequest(t *testing.T) {
 		isPayloadInvalid                       bool
 	}{
 		{
-			description: "HandlePipelineApproveOrRejectRunRequest: valid",
-			body: `{
-				"post_id": "mockPostID",
-				"channel_id": "mockChannelID",
-				"context": {
-				  "approvalId": "mockApproverID",
-				  "organization": "mockOrganization",
-				  "projectID": "mockProjectID",
-				  "requestType": "mockRequestType"
-				}
-			  }`,
+			description:                            "HandlePipelineApproveOrRejectRunRequest: valid",
+			body:                                   body,
 			updatePipelineRunApprovalRequestStatus: http.StatusOK,
 			statusCode:                             http.StatusOK,
 			getRunApprovalDetailsStatus:            http.StatusOK,
 		},
 		{
-			description: "HandlePipelineApproveOrRejectRunRequest: approved/rejected the request successfully but failed to update post",
-			body: `{
-				"post_id": "mockPostID",
-				"channel_id": "mockChannelID",
-				"context": {
-				  "approvalId": "mockApproverID",
-				  "organization": "mockOrganization",
-				  "projectID": "mockProjectID",
-				  "requestType": "mockRequestType"
-				}
-			  }`,
+			description:                            "HandlePipelineApproveOrRejectRunRequest: approved/rejected the request successfully but failed to update post",
+			body:                                   body,
```

Comment:
is body same for all test case if yes remove it and if its wrong for just one case have a bool valid body or not